raiden/network/proxies/token_network.py

```diff
@@ -1111,5 +1111,6 @@ def _check_channel_state_for_settle(self, participant1, participant2, channel_id
                 )
 
             raise RaidenRecoverableError(
-                'Channel cannot be settled before closing',
+                'Channel cannot be settled before closing. '
```

Comment:
This exception will be raised inside:
`elif channel_data.state == ChannelState.CLOSED:` which means that we know (for now) that the channel IS indeed closed. Maybe,
```
Settling this channel failed although the channel's current state is closed. Maybe it was already settled by the other participant?
```

Reply:
All we know is that we think the channel's state is closed. But sure I can reword it ... after tests run :)